engine/samples/collisions/main.cpp

```diff
@@ -63,13 +65,19 @@ int main()
                 .entity();
     });
 
-    cubos.startupSystem("create colliders").call([](State& state, Commands commands) {
+    cubos.startupSystem("create colliders").call([](State& state, Commands commands, Gravity& gravity) {
         state.a = commands.create()
                       .add(Collider{})
                       .add(BoxCollisionShape{})
                       .add(LocalToWorld{})
                       .add(Position{glm::vec3{0.0F, 0.0F, -2.0F}})
                       .add(Rotation{})
+                      .add(PreviousPosition{{0.0F, 0.0F, 0.0F}})
+                      .add(Velocity{.vec = {0.0F, 0.0F, 1.0F}})
+                      .add(Force{})
+                      .add(Impulse{})
+                      .add(Mass{.mass = 500.0F, .inverseMass = 1.0F / 500.0F})
+                      .add(AccumulatedCorrection{{0.0F, 0.0F, 0.0F}})
```

Comment:
This is very nice, it's much better than having to memorize all the components. This system should be added in the physics plugin itself right? As for the `PhysicsBundle` component, I think we should have mass, velocity, force and impulse in there, because we might want to spawn entities with forces being applied to them, what do you think?

Reply:
Yeah, LGTM